etl/steps/data/garden/regions/2023-01-01/regions.yml

```diff
@@ -1847,3 +1847,221 @@
   end_year: 1902
   successors:
     - "ZAF"
+
+# WHO regios
+- code: WHO_AMR
+  name: "WHO Americas region"
```

Comment:
I'm not sure about this name. This is the name that we will use everywhere, including charts. I'd suggest removing "region" from the names (since it's redundant). And I'm wondering whether the name should be, e.g. "Americas (WHO)", for consistency with our current naming conventions. What do you think?

Reply:
Good point, I think your suggestion to be consistent with our current naming conventions makes sense, I'll change it to that.